tools/runners/test262-harness.py

```diff
@@ -417,8 +418,8 @@ def dispose(self):
         try:
             self.close()
             os.unlink(self.name)
-        except OSError as exception:
-            logging.error("Error disposing temp file: %s", str(exception))
+        except OSError as os_error:
+            logging.error("Error disposing temp file: %s", str(os_error))
```

Comment:
why the str() around os_error? the %s in the format string will force conversion to string, exactly via str().